.changeset/weak-pugs-approve.md

```diff
@@ -0,0 +1,5 @@
+---
+'@contentful/f36-multiselect': patch
+---
+
+Fix for multiselect example in documentation
```

Comment:
Not sure if you need to release a new version since it's only a change on examples.
But up to you 😄